update_version.py

```diff
@@ -1,43 +1,21 @@
 #!/usr/bin/env python
-# Usage: ./update_version.py <MAJOR>.<MINOR>.<MICRO> [<RC version>]
-#
-# Example:
-# ./update_version.py 3.7.1 2
-#   => Version will become 3.7.1-rc-2 (beta)
-# ./update_version.py 3.7.1
-#   => Version will become 3.7.1 (stable)
+# Usage: ./update_version.py
 
 import datetime
+import json
 import re
 import sys
 from xml.dom import minidom
 
-if len(sys.argv) < 2 or len(sys.argv) > 3:
-  print("""
-[ERROR] Please specify a version.
-
-./update_version.py <MAJOR>.<MINOR>.<MICRO> [<RC version>]
-
-Example:
-./update_version.py 3.7.1 2
-""")
-  exit(1)
-
-NEW_VERSION = sys.argv[1]
+versions = json.load(open('versions.json', 'r'))
+protoc_version = versions['versions'][0]['protoc_version'].split('-')
+RC_VERSION = len(protoc_version) > 1 and int(protoc_version[1][2:]) or -1
+NEW_VERSION = protoc_version[0]
```

Comment:
Can we name this PROTOC_VERSION instead to keep it clear that it's not the NEW_VERSION for all languages?